Sources/Request/Helpers/MediaType.swift

```diff
@@ -12,4 +12,27 @@ import Foundation
 public enum MediaType: String {
```

Comment:
Maybe instead of including a second initializer for the `Form`s that uses a `String` (as in `init(mime: String, ...)`) we could add a `custom(String)` case to `MediaType` and conform to `CustomStringConvertible`. I think that'd make it a bit easier to use and cause a bit less duplication.